.github/workflows/snyk.yml

```diff
@@ -0,0 +1,54 @@
+name: Snyk SBOM CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  snyk_scans:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v3
+      - name: Run Snyk to check for vulnerabilities - sarif output
+        continue-on-error: true
+        uses: snyk/actions/golang@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          command: test
+          args:  --all-projects --sarif-file-output=${{ github.event.repository.name }}.sarif --strict-out-of-sync=false
```

Comment:
is there a reason we're running this twice with different output formats? (sarif and json)? 

is a single pass not sufficient?

Reply:
In my testing, you can only have one format type per run of Snyk unfortunately. We've had asks for both SARIF and JSON formats.